tests/cupy_tests/creation_tests/test_basic.py

```diff
@@ -35,6 +41,12 @@ def test_empty_int(self, xp, dtype, order):
         a.fill(0)
         return a
 
+    @testing.slow
+    def test_empty_huge_int(self):
+        a = cupy.empty(2 ** 31, dtype='b')
+        a.fill(0)
+        self.assertFalse(a.any())
+
```

Comment:
1. Test program uses about 16GB in int32 case. I think it is not acceptable.
2. OK, I will fix.